### PR TITLE
feat: update placeholders

### PR DESCRIPTION
**Pull request by kopkop123**

### CODE REVIEW

**Review by AlexKapic — APPROVED** [review]

File: frontend/src/components/settings/components/workspace-settings/workspace-settings.tsx [hunk 1]
```diff
@@ -148,7 +150,11 @@ export const WorkspaceSettings: React.FC = () => {
                       src={selectedImgURL || currentWorkspace?.logo}
                     />
                   ) : (
-                    <span className={styles.noLogo}>no logo</span>
+                    <img
+                      src={noImage}
+                      alt="Infostack Empty Logo"
+                      className={styles.noLogo}
+                    />
```
Comment (ovchynnikov) [inline frontend/src/components/settings/components/workspace-settings/workspace-settings.tsx]: `<i className="bi bi-card-image"></i>`

File: frontend/src/assets/img/no-image.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-card-image" viewBox="0 0 16 16">
+  <path d="M6.002 5.5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"/>
+  <path d="M1.5 2A1.5 1.5 0 0 0 0 3.5v9A1.5 1.5 0 0 0 1.5 14h13a1.5 1.5 0 0 0 1.5-1.5v-9A1.5 1.5 0 0 0 14.5 2h-13zm13 1a.5.5 0 0 1 .5.5v6l-3.775-1.947a.5.5 0 0 0-.577.093l-3.71 3.71-2.66-1.772a.5.5 0 0 0-.63.062L1.002 12v.54A.505.505 0 0 1 1 12.5v-9a.5.5 0 0 1 .5-.5h13z"/>
+</svg>
```
Comment (ovchynnikov) [inline frontend/src/assets/img/no-image.svg]: svg file is not required, we can use bootstrap icons with `<i>` tag

File: frontend/src/components/settings/components/workspace-settings/workspace-settings.tsx (hunk 1 above)
Comment (kopkop123) [reply frontend/src/components/settings/components/workspace-settings/workspace-settings.tsx]: fixed

File: frontend/src/assets/img/no-image.svg (hunk 2 above)
Comment (kopkop123) [reply frontend/src/assets/img/no-image.svg]: fixed

**Review by ovchynnikov — APPROVED** [review]

**Review by DeniGolf — APPROVED** [review]

**Pull request merged**